lib/components/modal/modal.test.ts

```diff
@@ -80,7 +80,11 @@ describe("modal", () => {
 
         await user.click(closeBtn);
 
-        await waitFor(() => expect(modal).not.to.be.visible);
+        // TODO: undestand why only in CI, for webkit the following assertion fails
```

Comment:
This is a odd one. For webkit and in in CI only the visible assertion does not seem to work correctly. Locally it works fine and it's only happening in this test. For now I decided to assert on the attribute aria-hidden that it used by the styles to determine the visibility.